lib/extract/netcdf/Makefile

```diff
@@ -0,0 +1,55 @@
+#------------------------------------------------------------------------------
+# BSD 2-Clause License
+# 
+# Copyright (c) 2018-2019, Science and Technology Facilities Council.
+# All rights reserved.
+# 
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+# 
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+# 
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+# 
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#------------------------------------------------------------------------------
+# Authors: J. Henrichs, Australian Bureau of Meteorology
+
+
+# This Makefile picks up the compiler to use plus any flags from
+# environment variables. e.g. to use gfortran:
+# 
+# export F90=mpif90
+# export F90FLAGS="-O3"
+# export AR=ar
+
```

Comment:
Maybe allow NETCDF_DIR to be picked-up from the environment too?

Reply:
(If I use system-installed netcdf libs then they don't follow the nice `/some/path/{include,lib}` structure so I'd need NETCDF_INCLUDE_DIR and NETCDF_LIB_DIR.)

Reply:
I've replaced all usage of paths with ``nc-config`` calls, so as long as this is in your path it should all be portable.

Reply:
As you have suggested, I've switched to nf-config, and referenced the bug in the README.